apps/trustlab/src/next-seo.config.js

```diff
@@ -0,0 +1,29 @@
+import site from "@/trustlab/utils/site";
+
+const config = {
+  // Disable indexing while in development
+  dangerouslySetAllPagesToNoFollow: true,
+  dangerouslySetAllPagesToNoIndex: true,
```

Comment:
Cool, but we should use env vars here... I think `codeforafrica` or `charterafrica` app does that already.